src/main/java/moze_intel/projecte/gameObjs/items/rings/ArchangelSmite.java

```diff
@@ -132,7 +132,7 @@ public void updateInPedestal(@Nonnull World world, @Nonnull BlockPos pos)
 			{
 				if (!world.getEntitiesWithinAABB(EntityLiving.class, tile.getEffectBounds()).isEmpty())
 				{
-					for (int i = 0; i < 3; i++)
+					for (int i = 0; i < 20; i++)
 					{
```

Comment:
This makes the Archangel more powerful in a pedestal?

Reply:
Its not good practice to sneak changes like this in with a big PR like this, without announcing them.